package.json

```diff
@@ -76,7 +76,7 @@
     "@microsoft/applicationinsights-analytics-js": "^3.0.5"
   },
   "publishConfig": {
-    "tag": "release2"
+    "tag": "angular14"
```

Comment:
Uppercase please `Angular14` so it matches the previous one
https://www.npmjs.com/package/@microsoft/applicationinsights-angularplugin-js?activeTab=versions